app-emulation/euae/euae-0.8.29.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="E-UAE Experimental Ubiquitous Amiga Emulator"
+DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an emulation core largely based on WinUAE. It attempts to bring many of the great features of WinUAE to non-Windows platforms. This version now finally has a name, E-UAE, since that's what everybody was calling it anyway. The 'E' can stand for anything you fancy. Experimental, extreme, exciting, egalitarian, eggplant, . . .
```

Comment:
Thanks for the link - I had been following [this guide](https://www.haiku-os.org/blog/humdinger/2016-05-28_contributing_pull_requests). I'll use feature branches in future.